roles/magnum/templates/etc/magnum/magnum.conf

```diff
@@ -0,0 +1,300 @@
+# value)
+#swarm_atomic_template_path = $pybasedir/templates/docker-swarm/swarm.yaml
+
+# Format string to use for swarm discovery url. Available values:
+# bay_id, bay_uuid. Example: "etcd://etcd.example.com/\%(bay_uuid)s"
+# (string value)
+#swarm_discovery_url_format = <None>
+
+# Indicates Swarm discovery should use public endpoint. (boolean
+# value)
+#public_swarm_discovery = true
+
+# Url for swarm public discovery endpoint. (string value)
+#public_swarm_discovery_url = https://discovery-stage.hub.docker.com/v1/clusters
+
+# Location of template to build a Mesos cluster on Ubuntu. (string
+# value)
+#mesos_ubuntu_template_path = $pybasedir/templates/heat-mesos/mesoscluster.yaml
+
+# Enabled bay definition entry points. (list value)
+#enabled_definitions = magnum_vm_atomic_k8s,magnum_vm_coreos_k8s,magnum_vm_atomic_swarm,magnum_vm_ubuntu_mesos
+
+
+[bay_heat]
+
+#
+# From magnum
+#
+
+# Number of attempts to query the Heat stack for finding out the
+# status of the created stack and getting template outputs.  This
+# value is ignored during bay creation if timeout is set as the poll
+# will continue until bay creation either ends or times out. (integer
+# value)
+#max_attempts = 2000
+
+# Sleep time interval between two attempts of querying the Heat stack.
+# This interval is in seconds. (integer value)
+#wait_interval = 1
+
+# The length of time to let bay creation continue.  This interval is
+# in minutes.  The default is no timeout. (integer value)
+#bay_create_timeout = <None>
+
+
+[conductor]
+
+#
+# From magnum
+#
+
+# The queue to add conductor tasks to. (string value)
+#topic = magnum-conductor
+
+# RPC timeout for the conductor liveness check that is used for bay
+# locking. (integer value)
+#conductor_life_check_timeout = 4
+
+
+[database]
+
+#
+# From magnum
+#
+
+connection=mysql://magnum:{{ secrets.db_password }}@{{ endpoints.db }}/magnum?charset=utf8
+
+[docker]
+
+#
+# From magnum
+#
+
+# Docker remote api version. Override it according to specific docker
+# api version in your environment. (string value)
+#docker_remote_api_version = 1.17
+
+# Default timeout in seconds for docker client operations. (integer
+# value)
+#default_timeout = 10
+
+# If set, ignore any SSL validation issues (boolean value)
+#api_insecure = false
+
+# Location of CA certificates file for securing docker api requests
+# (tlscacert). (string value)
+#ca_file = <None>
+
+# Location of TLS certificate file for securing docker api requests
+# (tlscert). (string value)
+#cert_file = <None>
+
+# Location of TLS private key file for securing docker api requests
+# (tlskey). (string value)
+#key_file = <None>
+
+
+[glance_client]
+
+#
+# From magnum
+#
+
+# Region in Identity service catalog to use for communication with the
+# OpenStack service. (string value)
+#region_name = <None>
+
+# Type of endpoint in Identity service catalog to use for
+# communication with the OpenStack service. (string value)
+#endpoint_type = publicURL
```

Comment:
This would be a good place to start hitting the internalURL endpoint type, or at least commenting somewhere that we can frob it when we're ready to separate customer calls from service calls.